app/src/main/res/layout/view_multi_action_add.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should this be wrapped in a `MaterialCardView`? I was thinking about the `background`-attribute, I think the ripple comes automatically when you make a `MaterialCardView` clickable 🙂

Reply:
Good idea